docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -1646,11 +1646,11 @@ commands. Each command has the following format:
       Insert length number of literals, with the following format:
 
          Block type code for next literal block type, appears
-            only if NBLTYPESI >= 2 and the previous literal
+            only if NBLTYPESL >= 2 and the previous literal
```

Comment:
@dsnet Mistakes happen. It's very hard to write something like this, where potentially every single word carries actual information. And then it gets even harder, if the goal is to convey the correct information, but in a way that's easy to understand.

For what it's worth, I found all your changes very helpful, and had actually written a fairly long comment on why I found it necessary to mention that duplicate symbols in simple prefix codes are to be considered invalid, when I then noticed that in the newest version, you had already addressed it.

It's a process, and I believe it's going well overall. ;)